modules/vf-graphql-holochain/mutations/proposedTo.ts

```diff
@@ -5,26 +5,33 @@
  * @since:   2019-09-12
  */
 
-import { DNAIdMappings } from '../types'
+import { AgentAddress, ByRevision, DNAIdMappings, ProposalAddress } from '../types'
 import { mapZomeFn } from '../connection'
 import { deleteHandler } from './'
 
 import {
   ProposedToResponse,
 } from '@valueflows/vf-graphql'
 
+export interface CreateArgs {
+  proposedTo: {
+    proposed: AgentAddress,
+    publishes: ProposalAddress,
+    reciprocal: boolean,
+  },
+}
 export type createHandler = (root: any, args) => Promise<ProposedToResponse>
 
 export default (dnaConfig: DNAIdMappings, conductorUri: string) => {
-  const runCreate = mapZomeFn(dnaConfig, conductorUri, 'proposal', 'proposed_to', 'create_proposed_to')
-  const runDelete = mapZomeFn(dnaConfig, conductorUri, 'proposal', 'proposed_to', 'delete_proposed_to')
+  const runCreate = mapZomeFn<CreateArgs, ProposedToResponse>(dnaConfig, conductorUri, 'proposal', 'proposed_to', 'create_proposed_to')
+  const runDelete = mapZomeFn<ByRevision, boolean>(dnaConfig, conductorUri, 'proposal', 'proposed_to', 'delete_proposed_to')
 
   const proposeTo: createHandler = async (root, args) => {
-    return runCreate({ proposed_to: args })
+    return runCreate(args)
```

Comment:
you also changed the input type. Make sure that serde rename all is right on the related zome struct